smurfs/src/components/SmurfList.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import SmurfCard from './SmurfCard';
+
+const SmurfList = props => {
+    console.log(props.smurfs);
+    return (
+        <div>
+            {props.smurfs.map(smurf => (
+                <SmurfCard smurf={smurf} key={smurf.name} />
```

Comment:
`smurf.id` would make a better key since the server provides one for each smurf
```suggestion
                <SmurfCard smurf={smurf} key={smurf.id} />
```